osquery/events/linux/auditdfim.h

```diff
@@ -0,0 +1,129 @@
+#pragma once
+
+#include "osquery/events/linux/auditdnetlink.h"
+#include <osquery/events.h>
+
+namespace osquery {
+
+/// Syscall event descriptor
+struct SyscallEvent final {
+  /// Event type
+  enum class Type {
+    Execve,
+    Exit,
+    Exit_group,
+    Open,
+    Openat,
+    Name_to_handle_at,
+    Open_by_handle_at,
+    Close,
+    Dup,
+    Read,
+    Creat,
+    Mknod,
+    Mknodat,
+    Write,
+    Mmap,
+    Invalid
+  };
+
+  //
+  // Populated from the AUDIT_SYSCALL event record
```

Comment:
If you need multi-line comments use the long-form docblock:
```
/**
 * @brief Here is a summary.
 *
 * Some more stuff yay!
 */
```

Only use the `///` short-form for documenting a member or method that only needs a 1-liner.